README.md

```diff
@@ -91,10 +91,26 @@ export default () => (
 
 ```
 
+```jsx
+// pages/index.js
+import {Link} from '../routes'
+
+export default () => (
+  <div>
+    <div>Welcome to next.js!</div>
+    <Link path='/some-path-conform-to-a-route-patter'>
+      <a>Hello world</a>
+    </Link>
+  </div>
+)
+
+```
+
 API: `<Link route="name" params={params}>...</Link>`
 
 - `route` - Name of a route
 - `params` - Optional parameters for the route URL
+- `path`- A path supposed to follow a route pattern. From the path the right route and params are found. NOTE: `path`and `route` exclusive and if present `route` takes precedence over `path`.
```

Comment:
Please clarify the last part/intention, the implementation seems that `path` takes precedence.